docs/src/layouts/MainLayout.vue

```diff
@@ -93,28 +20,28 @@
             v-if="showFooterToolbar(footerIndex)"
             :label="label"
             :to="to"
-            color="lp-light"
+            color="white-54"
```

Comment:
This color and the the white background on which it is placed will cause text to disappear

Reply:
If I'm not wrong, the whole footer toolbar is wrong in terms of content. We can probably use a better way to reuse the existing header and just make it change color scheme. Let's do this in a following PR, I'll just fix the color for now